### ISSUE TITLE
Incorrect recalc availableWidth

### ISSUE THREAD

**Issue by dudev** [bpampuch/pdfmake#858]
Right column on the next page

![image](https://cloud.githubusercontent.com/assets/7987299/22732752/35564042-ee11-11e6-90f0-ff057a207a4d.png)

In function DocumentContext.prototype.moveToNextPage (DocumentContext.js on 204 line):
`this.pageSnapshot().availableWidth = pageSize.width - this.x - this.pageMargins.right;`

where this.x = 302.64

My code:

```
docDefinition.content.push({
    columns: [
        {
            text: `long text`,
            width: "50%",
        },
        {
            text: `too long text`,
            width: "50%",
        }
    ],
    columnGap: 10
});
docDefinition.content.push({
    text: "Big5",
    fontSize: 24,
    bold: true,
    alignment: "center",
    margin: [0, 20, 0, 0]
});
```

Next block have availableWidth=252.64, but this is incorrect.

**Comment by liborm85** [comment]
Please sample docDefinition for reproducing problem, which working on http://pdfmake.org/playground.html. Thanks.

**Comment by dudev** [comment]
[dd.txt](https://github.com/bpampuch/pdfmake/files/760405/dd.txt)

**Comment by dudev** [comment]
http://pdfmake.org/playground.html
Look at COLUMNS

Why last string height is 3 lines? availableWidth too?

![image](https://cloud.githubusercontent.com/assets/7987299/22734613/246d5426-ee18-11e6-8c28-8b3f8887ae9a.png)

**Comment by liborm85** [comment]
Broke it this issue fix: https://github.com/bpampuch/pdfmake/issues/840 I'll try to fix it.

**Comment by liborm85** [comment]
Fix for #840 reverted and fixed by commit https://github.com/bpampuch/pdfmake/commit/bbc8e71d5783ca42f420d04ae44f1ecf39d3935e
